ReleaseNotes.md

```diff
@@ -8,6 +8,38 @@ information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeo
 additional questions or comments.
 
 ## Release History
+### 06/30/2023 -
+##### Version microsoft/oms:3.1.10 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:3.1.10 (linux)
+##### Version microsoft/oms:win-3.1.10 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.10 (windows)
+##### Current dependencies
+- Linux
+  - [CBL-Mariner 2.0.20230526](https://github.com/microsoft/CBL-Mariner/releases/tag/2.0.20230526-2.0)
+  - Golang - 1.20.3
+  - Ruby - 3.1.3
+  - MDSD - 1.26.1
+  - Telegraf - 1.26.0
+  - Fluent-bit - 2.0.9
+  - Fluentd - 1.14.6
+- Windows
+  - Ruby - 3.1.1
+  - Fluent-bit - 2.0.14
+  - Telegraf - 1.24.2
+  - Fluentd - 1.14.6
+  - Windows AMA - 46.3.2
+  - Golang - 1.20.3
+##### Code change log
+- Common
+  - Add Containerlogv2 support in dataCollectionSettings
```

Comment:
I think Amol mentioned some tool to update the change log. Were you able to try that?

Reply:
good point  let me try that

Reply:
The generated release notes will be something like this, what you guys think: ## What's Changed
* Gangams/fix typo envvar name by @ganga1980 in https://github.com/microsoft/Docker-Provider/pull/995
* Fix weird non-space whitespace by @TeamDman in https://github.com/microsoft/Docker-Provider/pull/996
* Troubleshoot doc update for MSI GA by @wanlonghenry in https://github.com/microsoft/Docker-Provider/pull/991
* fix fluentd crash in windows by @ganga1980 in https://github.com/microsoft/Docker-Provider/pull/1001
* Longw/window vul by @wanlonghenry in https://github.com/microsoft/Docker-Provider/pull/998
* Longw/containerlogv2 dcr  by @wanlonghenry in https://github.com/microsoft/Docker-Provider/pull/994
* Windows spin-wait thread pool issue by @jatakiajanvi12 in https://github.com/microsoft/Docker-Provider/pull/1002
* Longw/containerlogv2 dcr merge by @wanlonghenry in https://github.com/microsoft/Docker-Provider/pull/1004
* upgrade fluent-bit to 2.0.14 by @wanlonghenry in https://github.com/microsoft/Docker-Provider/pull/1005

## New Contributors
* @TeamDman made their first contribution in https://github.com/microsoft/Docker-Provider/pull/996

**Full Changelog**: https://github.com/microsoft/Docker-Provider/compare/3.1.9...3.1.10-test

Reply:
yes we can add it in tag info as well as release notes